scriptoocloud/src/main/java/edu/asu/diging/scriptoocloud/web/validation/RepositoryUrlConstraintValidator.java

```diff
@@ -0,0 +1,28 @@
+package edu.asu.diging.scriptoocloud.web.validation;
+
+import java.net.URL;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class RepositoryUrlConstraintValidator implements ConstraintValidator<RepositoryUrlConstraint, String> {
+
+    private Logger logger = LoggerFactory.getLogger(getClass());
+
+    @Override
+    public void initialize(RepositoryUrlConstraint constraintAnnotation) {}
+
+    public boolean isValid(String urlField, ConstraintValidatorContext context) {
+        boolean valid = true;
+        try {
+            URL url = new URL(urlField);
+        } catch(final Exception ignore){
+            logger.error("Malformed URL provided");
```

Comment:
log stack trace and make it warn or info. It's a validator, so it's job is to find non-valid inputs. It doesn't need to log in error level, but when you log it you want the stacktrace in case some comes and asks "what's wrong with this".